src/main/java/alice/task/Deadline.java

```diff
@@ -0,0 +1,86 @@
+package alice.task;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+import alice.AliceException;
+
+/**
+ * Represents a task with a deadline.
+ */
+public class Deadline extends Task {
+    private static final DateTimeFormatter D_DATETIME_FORMAT = DateTimeFormatter
+            .ofPattern("EEEE, MMM dd uuuu, ha");
+    private static final DateTimeFormatter D_DATE_FORMAT = DateTimeFormatter
+            .ofPattern("EEEE, MMM dd uuuu");
+
+    private final LocalDateTime by;
+    private final boolean includesTime;
```

Comment:
Perhaps a more intuitive variable name to sound more like a boolean?

Reply:
Agree. Updated on my end. Thanks for the input.